app/models/project_wiki.rb

```diff
@@ -102,8 +102,8 @@ def delete_page(page, message = nil)
   end
 
   def page_title_and_dir(title)
-    title_array =  title.split("/")
-    title = title_array.pop
+    title_array = title.split("/")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.